docs/data-sources/load_balancer.md

```diff
@@ -21,7 +27,7 @@ description: |-
 
 ### Read-Only
 
-- `forwarding_rules` (List of Object) The rules that control which traffic the load balancer will forward to servers in the pool. Leave null to accept a default "HTTP" only forwarding rule. (see [below for nested schema](#nestedatt--forwarding_rules))
+- `forwarding_rules` (Attributes List) The rules that control which traffic the load balancer will forward to servers in the pool. Leave null to accept a default "HTTP" only forwarding rule. (see [below for nested schema](#nestedatt--forwarding_rules))
```

Comment:
Fix description to remove "Leave null to accept a default "HTTP" only forwarding rule."